plugins/main/public/utils/applications.ts

```diff
@@ -232,31 +232,6 @@ export const mitreAttack = {
     }`,
 };
 
-export const virustotal = {
```

Comment:
**nitpick**: I think this variable is used (title at least) when you restored this: https://github.com/wazuh/wazuh-dashboard-plugins/pull/7038/commits/52565172b44565c5d4baf045c642d17557c968c4.

You could remove the variable and add the string in the expected view if we consider or remove it.